src/editors/containers/VideoEditor/components/VideoEditorModal.jsx

```diff
@@ -16,21 +16,31 @@ export const hooks = {
 };
 
 const VideoEditorModal = ({
-  isOpen,
   close,
+  error,
+  isOpen,
 }) => {
   const dispatch = useDispatch();
   module.hooks.initialize(dispatch);
   return (
-    <VideoSettingsModal {...{ isOpen, close }} />
+    <VideoSettingsModal {...{ close, error, isOpen }} />
   );
   // TODO: add logic to show SelectVideoModal if no selection
 };
 
 VideoEditorModal.defaultProps = {
+  error: {
```

Comment:
Have you thought about other data structures for use to represent errors?

Reply:
I'm thinking about how we might have multiple failed validations for a single widget?

Reply:
I should have mentioned but I imagined the data structure to look like the below. I'm open to suggestions on other ways to do this. (I'll update the top level comment to describe this.)
```
errror: {
  duration: {
    [fieldName]: "error text",
    [fieldName]: "error text",
  },
  handout: {
    ...
  },
}
```

Reply:
Ok yeah So that makes more sense -- we just want to be sure we aren't over-specifying the shape in places we don't have to.